### PR TITLE
update github issue templates

### PR DESCRIPTION
**Pull request by MonsieurNicolas**
Adds links to community from the issue type selector

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 1]
```diff
@@ -3,31 +3,38 @@ name: Documentation
...
+If you can't find an answer to your question you can:
+* submit a question to [Stellar's Stack Exchange][2].
+* or ask one of [Stellar's communities][3].
 
 [1]: https://www.stellar.org/developers/
 [2]: https://stellar.stackexchange.com/
-[3]: https://www.stellar.org/community
+[3]: https://www.stellar.org/community/#communities
+
+### I'm fairly certain it's a bug.
```
Comment (marta-lokhova) [inline .github/ISSUE_TEMPLATE/documentation.md]: is this paragraph here intentionally?

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 2]
```diff
@@ -51,8 +50,8 @@ flag or HTTP command added to stellar-core), this is the place.
 *Be descriptive, including the interface you'd like to see, as well as any suggestions you may have
 with regard to its implementation.*
 
-### Is your feature request related to a problem? Please describe.
-*A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]*
+### Explain why this feature feature is important
```
Comment (jonjove) [inline .github/ISSUE_TEMPLATE/feature-request.md]: Nitpick: "feature feature" -> "feature"

File: .github/ISSUE_TEMPLATE/documentation.md (hunk 1 above)
Comment (MonsieurNicolas) [reply .github/ISSUE_TEMPLATE/documentation.md]: copy paste/error, fixed

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 2 above)
Comment (MonsieurNicolas) [reply .github/ISSUE_TEMPLATE/feature-request.md]: fixed

**Comment by marta-lokhova** [comment]
r+ e7870a68cb41c54947979e5cca87ede625a72156

**Pull request merged**